src/random.c

```diff
@@ -176,6 +176,7 @@ random_get_uint32 (Random       *random)
 
     return dest;
 }
+
```

Comment:
whitespace

Reply:
Done.  Removed line between functions to make it less readable.